iChef-WebUi/src/app/styles/variables.scss

```diff
@@ -5,10 +5,14 @@ App color palette
  */
 
 $primary-color: #E98074;
+$primary-color2: rgba($color:#E98074, $alpha:0.75);
```

Comment:
I don't think that 'primary-color2' is a good name, maybe rename it to 'primary-color-faded' or other specific term. 
Update: You used the term 'transparent' below, that could work as well.